Jenkinsfile

```diff
@@ -106,10 +106,18 @@ podTemplate(
             println("This version: " + currentVersion)
             def versionExists = sh(returnStdout: true, script: 'pipenv run python3 cognite/client/utils/_version_checker.py -p cognite-sdk -v ' + currentVersion).trim()
             println("Version Exists: " + versionExists)
+
             if (env.BRANCH_NAME == 'master' && versionExists == 'no') {
                 stage('Release') {
                     sh("pipenv run twine upload --config-file /pypi/.pypirc dist/*")
                 }
+                stage('Update code snippets on service-contracts') {
+                    sh("pip3 install requests")
```

Comment:
Requests should already be installed.

Reply:
you sure? it did fail for me last time though

Reply:
ok, we realised that I need `pipenv`